### PR TITLE
Fixes backslot not counting towards assess_threat

### LINKED ISSUE #38033: Security bots don't increase threat level for weapons worn on back
Sec bots give +2 threat level for weapons worn on the belt slot (+4 for inhand), but the back slot is ignored completely:

https://github.com/tgstation/tgstation/blob/ba1030f8038f07bfba17d4b2887c584d09b8c9fe/code/modules/mob/living/carbon/human/human.dm#L571-L578

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #38033

🆑 ShizCalev
fix: Weapons slung on one's back will now count towards beepsky's threat assessment.
/🆑

### CODE REVIEW

File: code/modules/mob/living/carbon/monkey/monkey.dm [hunk 1]
```diff
@@ -137,6 +137,8 @@
 		for(var/obj/item/I in held_items)
 			if(weaponcheck.Invoke(I))
 				threatcount += 4
+			if(weaponcheck.Invoke(back))
```
Comment (SpaceManiac) [inline code/modules/mob/living/carbon/monkey/monkey.dm]: Wrong level of indentation

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Review by SpaceManiac — DISMISSED** [review]

**Comment by 81Denton** [comment]
This doesn't seem to work, the secbot I tested this with didn't increase its threatlevel for an egun worn on the back slot. Belt/inhand worked normally like before.

**Comment by ShizCalev** [comment]
@81Denton I tested it and it works correctly. 

Mind you having a weapon on your back or belt only adds +2 to your threat assessment, so you'd need to have either two weapons (both back and belt), or be set to paroled for it to meet the >=4 requirement to be chased.

**Comment by 81Denton** [comment]
That's odd, I checked out your branch and then checked the bot's threarlevel var via VV. Wearing an egun on that slot wouldn't change the value, only belt/inhand.

**Comment by ShizCalev** [comment]
Just tested it again to make sure, it _is_ working.

**Comment by 81Denton** [comment]
Oh, sorry about that. Not sure what I did wrong.

**Pull request merged**